test/test_dtensor.py

```diff
@@ -171,37 +168,28 @@ def test_dtensor_fp8_autograd(mesh: DeviceMesh, size=16):
 
 
 def _test_fp8_mlp_tensor_parallelism_base(
-    mesh: DeviceMesh, size=16, compile: bool = False, use_float8_linear: bool = False
+    mesh: DeviceMesh, size=16, compile: bool = False
 ):
     device = mesh.device_type
-    # TODO(future): delete Float8DynamicLinear from this test once all the
-    # code is unified
-    float8_cls = Float8Linear if use_float8_linear else Float8DynamicLinear
-    extra_kwargs = {}
-    if use_float8_linear:
-        # For now, just use Float8Linear with dynamic scaling, which is the
-        # same behavior as Float8Linear.
-        # TODO(future): add support for float8 all-gather with delayed scaling
-        # for activations and gradients.
-        extra_kwargs = {
-            "scaling_type_x": TensorScalingType.DYNAMIC,
-            "scaling_type_w": TensorScalingType.DYNAMIC,
-            "scaling_type_dL_dY": TensorScalingType.DYNAMIC,
-        }
+    # For now, just use Float8Linear with dynamic scaling, which is the
+    # same behavior as Float8Linear.
```

Comment:
Float8Dynamic ? But also its probably to to just say, only supports dynamic scaling for all 3 tensors, x, w, dl_dY

Reply:
agreed, let me fix in a future PR to speed up landing this, since this is a minor point.